hphp/runtime/vm/jit/unwind-itanium.cpp

```diff
@@ -358,7 +365,24 @@ void write_tc_cie(EHFrameWriter& ehfw) {
 
   // The following calculation is related to the "top" of the record.  There is
   // an implicit -8 factor as defined by EHFrameWriter::m_cie.data_align.
+
+#if defined(__powerpc64__)
```

Comment:
Yes, this makes sense to me.  I think it's perfectly fine to have this conditional on architecture—but could you make this one into a switch or `if ()` instead of guarding on `#ifdef`s?